qiskit_experiments/test/Fine Calibration Tutorial.rst

```diff
@@ -0,0 +1,154 @@
+
+        inst_map.add(name, (qubit,), sched)
+
+We now take the ideal x pulse amplitude reported by the backend and 
+add/subtract a 2% over/underrotation to it by scaling the ideal amplitude and see 
+if the experiment can detect this over/underrotation. We replace the default X pulse 
+in the instruction schedule map with this over/underrotated pulse.
+
+.. jupyter-execute::
+    ideal_amp = x_pulse.amp
+    over_amp = ideal_amp*1.02
+    under_amp = ideal_amp*0.98
+    print(f"The reported amplitude of the X pulse is {ideal_amp:.4f} which we set as ideal_amp. we use {over_amp:.4f} amplitude for overroation pulse and {under_amp:.4f} for underrotation pulse")
+
+-----------------------------------------------------------------------------------
+Detecting an over & under rotated pulse
+-----------------------------------------------------------------------------------
+
+.. jupyter-execute::
+    # build the over rotated pulse and add it to the instruction schedule map
+    with pulse.build(backend=backend, name="x") as x_over:
+        pulse.play(pulse.Drag(x_pulse.duration, over_amp, x_pulse.sigma, x_pulse.beta), d0)
+    inst_map.add("x", (qubit,), x_over)
+
+    # do the experiment
+    overamp_cal = FineXAmplitude(qubit, backend=backend)
+    overamp_cal.set_traspile_options(inst_map=inst_map)
+    exp_data_over = overamp_cal.run(backend)
+    exp_data_over.figure(0)
+
+.. jupyter-execute::
+    # build the under rotated pulse and add it to the instruction schedule map
+    with pulse.build(backend=backend, name="x") as x_under:
+        pulse.play(pulse.Drag(x_pulse.duration, under_amp, x_pulse.sigma, x_pulse.beta), d0)
+    inst_map.add("x", (qubit,), x_under)
+
+    # do the experiment
+    underamp_cal = FineXAmplitude(qubit, backend=backend)
+    underamp_cal.set_traspile_options(inst_map=inst_map)
+    exp_data_under = underamp_cal.run(backend)
+    exp_data_under.figure(0)
+
+.. jupyter-execute::
+    # analyze the results
+    target_angle = np.pi
+    dtheta_over = exp_data_over.analysis_results("d_theta").value.nominal_value
+    scale_over = target_angle / (target_angle + dtheta_over)
+    dtheta_under = exp_data_under.analysis_results("d_theta").value.nominal_value
+    scale_under = target_angle / (target_angle + dtheta_under)
+    print(f"The ideal angle is {target_angle:.2f} rad. We measured a deviation of {dtheta_over:.3f} rad in over-rotated pulse case.")
+    print(f"Thus, scale the {over_amp:.4f} pulse amplitude by {scale_over:.3f} to obtain {over_amp*scale_over:.5f}.")
+    print(f"On the other hand, we measued a deviation of {dtheta_under:.3f} rad in under-rotated pulse case.")
+    print(f"Thus, scale the {under_amp:.4f} pulse amplitude by {scale_under:.3f} to obtain {under_amp*scale_under:.5f}.")
+
+-----------------------------------------------------------------------------------
+Analyzing a pi/2 pulse
+-----------------------------------------------------------------------------------
+.. jupyter-execute::
```

Comment:
This is good but it needs some text. You point out some of the differences with calibrating an X gate. Namely, that with the SX gate we simply need to chose the right number of repetitions to always have the ideal population land on the equator of the Bloch sphere. Also, there should be on repetition done with two sx gates which allows us to calibrate the amplitude of the ping-pong pattern. This helps take measurement errors into account.